devtools/create_project/msvc.cpp

```diff
@@ -141,18 +123,12 @@ void MSVCProvider::createOtherBuildFiles(const BuildSetup &setup) {
 
 	// Create the configuration property files (for Debug and Release with 32 and 64bits versions)
 	// Note: we use the debug properties for the analysis configuration
-	createBuildProp(setup, true, ARCH_AMD64, "Release");
-	createBuildProp(setup, true, ARCH_X86, "Release");
-	createBuildProp(setup, true, ARCH_ARM64, "Release");
-	createBuildProp(setup, false, ARCH_AMD64, "Debug");
```

Comment:
These were initially called out explicitly because I wasn't sure enough it's safe to fold them

Reply:
I couldn't find any issues during my testing, should be easy enough to revert if any arise.